phalcon/Mvc/Model.zep

```diff
@@ -3769,6 +3784,19 @@ abstract class Model extends AbstractInjectionAware implements EntityInterface,
      *
      * @param string|array table
      */
+     protected function doLowUpdate(<MetaDataInterface> metaData, <AdapterInterface> connection, var table) -> bool
+     {
+         return this->_doLowUpdate(metaData, connection, table);
```

Comment:
Here and all other places: As a far as doLowUpdate used by default could you please invert this logic wrap doLowUpdate inside _doLowUpdate.